src/calliope/config/model_def_schema.yaml

```diff
@@ -675,7 +675,7 @@ properties:
             x-type: str
             title: Sink unit
             description: >-
-              Sets the unit of `Sink` to either `absolute` x-unit: energy), `per_area` x-unit: energy/area), or `per_cap` x-unit: energy/power).
+              Sets the unit of `Sink` to either `absolute` (e.g. kWh), `per_area` (e.g. kWh/m2), or `per_cap` (e.g. kWh/kW).
```

Comment:
I'd stick with `energy/power/area`. We have found in the past that by defining specific scales of units - even in examples - users can confuse that with _the_ scale that must be used.

Reply:
That's what it currently looks like
![image](https://github.com/user-attachments/assets/09fd2de3-890d-455e-bb6e-b11d1d3ab221)

and that's what (as an example) `source_unit` looks like
![image](https://github.com/user-attachments/assets/f15b649f-e564-44e8-8992-ae078719581a)

I assumed the second one was what it should look like. Just to clarify, do you mean both should read `... (energy) ...` (or `... (x-unit: energy) ...`?) instead?

Reply:
It should probably be `... (unit: energy) ...`. Looks like the `x-unit` bit has probably been caused by a replace-all.

Reply:
Thanks! Updated that (also accordingly for `source_unit`).